mkdocs.yml

```diff
@@ -36,7 +36,8 @@ theme:
 
   features:
     - navigation.top # Add back to top button
-    - toc.integrate
+    - toc.integrate # Integrate table of contents into left sidebar
```

Comment:
I've added some comments that explain the features.

Do you think this is helpful when reading the config file, or do you think this is making things messy?

Reply:
Maybe also add a link to the docs?

Reply:
Example of a link:

```yaml
    - toc.integrate # Integrate table of contents into left sidebar https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#integrated-table-of-contents
```

You can see that the link is quite long, so I don't think we want to put the link inline with the config option?

---

We could put a link to the entry point of the `mkdocs-material` documentation at the top of the `mkdocs.yml` file at least, that would point us in the right direction if we need to refer to the docs later on.

```yaml
# Documentation for Material for MkDocs: https://squidfunk.github.io/mkdocs-material/
```

Reply:
We can splitting the comments above the setting over multiple lines, which should make the lines shorter.

I'm pretty sure I don't remember these settings in a few months. 😅

Reply:
It's probably best if I do this in one go for all config options. This way we can review it in one go, instead of small bits.

I've created a new issue to track this comment rework. [^issue]

[^issue]: https://github.com/renovatebot/renovatebot.github.io/issues/129